test/test_selector_test.rb

```diff
@@ -21,6 +21,15 @@ def test_selects_tests
       assert_equal ['test/fizz_test.rb'], @selector.tests.to_a
     end
 
+    def test_selects_tests_from_current_working_tree
+      @repo.checkout('change_fizz')
+      # Change buzz too
+      fizzbuzz_file = "#{@repo.workdir}/lib/fizzbuzz.rb"
+      File.write(fizzbuzz_file, File.read(fizzbuzz_file).gsub(/"buzz"$/, '"bar"'))
+      selector = TTNT::TestSelector.new(@repo, nil, @test_files)
+      assert_equal selector.select_tests!, Set.new(['test/fizz_test.rb', 'test/buzz_test.rb'])
```

Comment:
Oops... :scream: Thank you for noticing! Fixed: https://github.com/Genki-S/ttnt/commit/f2d6440884dfe014edf5c2565786f77bead1ebe4